### ISSUE TITLE
New API-endpoint for european region 

### ISSUE THREAD

**Issue by drummerroma** [readysteady/mailgunner#19]
Mailgun introduced a new api endpoint for sending from European regions. To keep api calls working with domains in the new region the api endpoint should be configurabel.

The new api endpoint is:
api.eu.mailgun.net

**Comment by timcraft** [comment]
Thanks for the heads up, I'll take a look when I have time to do a new release.

**Comment by timcraft** [comment]
@drummerroma I've added an `api_host` option in dad1298 for this (available in v2.6.0)

No need for the whitespace changes and I'd rather not make up any non standard environment variables like MAILGUN_ENDPOINT in the library. If you need that you can do this:

```ruby
mailgun = Mailgunner::Client.new({
  # ...
  api_host: ENV.fetch('MAILGUN_ENDPOINT')
})
```

Thanks for the heads up!
